packages/flutter_tools/test/commands.shard/hermetic/devices_test.dart

```diff
@@ -36,6 +37,18 @@ void main() {
       DeviceManager: () => DeviceManager(),
       ProcessManager: () => MockProcessManager(),
     });
+
+    testUsingContext('Outputs parsable JSON with --machine flag', () async {
+      final DevicesCommand command = DevicesCommand();
+      await createTestCommandRunner(command).run(<String>['devices', '--machine']);
+      expect(
+        json.decode(testLogger.statusText),
+        mockDevices.map((MockDeviceJson d) => d.json)
```

Comment:
It would be better if you could define the values you are asserting on within the test. As is it is difficult to tell what is doing.

Reply:
@jonahwilliams I use these values in two different tests and that's why I moved them into common `test/src/mock_devices.dart`. Should I just blindly copy-paste here or just providing a comment is enough? For me, it just takes pressing F12 on `MockDeviceJson` to jump to that file and see everything.

Reply:
Ahh I see you have this already, my b